src/lib/mina_base/signed_command_intf.ml

```diff
@@ -236,11 +236,15 @@ module type Full = sig
 
     module V2 : sig
       type t =
-        ( Payload.Stable.V2.t
-        , Public_key.Stable.V1.t
-        , Signature.Stable.V1.t )
+        ( Payload.Stable.Latest.t
```

Comment:
I don't think we should do this. A versioned type should be built from other specific versioned types. If those other types add new versions, this type will change.

It is true that the new version linter will detect such changes, but still.

Reply:
It is also true that this is a signature, not an implementation. But still ...

Reply:
I changed these to get around the ppx_version warning, but have since disabled it, so I can undo all of these changes to refer to the latest versions.